packages/ssr/package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "@supabase/ssr",
+	"version": "0.0.0",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"publishConfig": {
+		"access": "public"
+	},
+	"files": [
+		"dist"
+	],
+	"scripts": {
+		"lint": "tsc",
+		"build": "tsup"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/supabase/auth-helpers.git"
+	},
+	"keywords": [
+		"Supabase",
+		"Auth",
+		"Svelte Kit",
+		"Svelte"
+	],
+	"author": "Supabase",
+	"license": "MIT",
+	"bugs": {
+		"url": "https://github.com/supabase/auth-helpers/issues"
+	},
+	"homepage": "https://github.com/supabase/auth-helpers#readme",
+	"devDependencies": {
+		"@supabase/supabase-js": "2.33.1",
+		"@types/cookie": "^0.5.1",
+		"@types/ramda": "^0.29.3",
+		"cookie": "^0.5.0",
+		"ramda": "^0.29.0",
```

Comment:
these ones seem like full depenencies (not devDependencies)? Perhaps not, just flagging in case

Reply:
As full dependencies they will get installed as individual packages, as dev dependencies the transpiler will treeshake them and include only the code from these libraries that are being used by this package.

Reply:
Although this makes a huge difference in bundle size. Will move these to `dependencies` instead of `devDependencies`.